pkg/model/form_submission.go

```diff
@@ -0,0 +1,81 @@
+package model
+
+import (
+	"fmt"
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
+// this is what we expect for input for a form submission
+type FormSubmissionAnswerInput struct {
+	WidgetId int64       `json:"widget_id"`
+	Answer   interface{} `json:"answer"`
+}
+
+type FormSubmissionInput struct {
+	FormId  string                      `jsont:"form_id"`
+	Status  string                      `json:"status" bson:"status"`
```

Comment:
Yes, the front end hasn't defined any enum style set of valid values, so at the moment Pillar has no opinion on what is set there other than that it is a string.